contracts/governance/GovernorAlpha.sol

```diff
@@ -0,0 +1,348 @@
+// Redistribution and use in source and binary forms, with or without modification, are permitted provided that the following conditions are met:
+// 1. Redistributions of source code must retain the above copyright notice, this list of conditions and the following disclaimer.
+// 2. Redistributions in binary form must reproduce the above copyright notice, this list of conditions and the following disclaimer in the documentation and/or other materials provided with the distribution.
+// 3. Neither the name of the copyright holder nor the names of its contributors may be used to endorse or promote products derived from this software without specific prior written permission.
+// THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS" AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+//
+// Ctrl+f for OLD to see all the modifications.
+
+// OLD: pragma solidity ^0.5.16;
+pragma solidity ^0.6.10;
+pragma experimental ABIEncoderV2;
+
+contract GovernorAlpha {
+    /// @notice The name of this contract
+    // OLD: string public constant name = "Compound Governor Alpha";
+    string public constant name = "TrustToken Governor Alpha";
+
+    // @notice The number of votes in support of a proposal required in order for a quorum to be reached and for a vote to succeed
+    // OLD: function quorumVotes() public pure returns (uint) { return 400000e18; } // 400,000 = 4% of Comp
+    function quorumVotes() public pure returns (uint) { return 58000000e8; } // 58,000,000 = 4% of Tru
+
+    // @notice The number of votes required in order for a voter to become a proposer
+    // OLD: function proposalThreshold() public pure returns (uint) { return 100000e18; } // 100,000 = 1% of Comp
+    function proposalThreshold() public pure returns (uint) { return 14500000e8; } // 14,500,000 = 1% of TRU
+
+    // @notice The maximum number of actions that can be included in a proposal
+    function proposalMaxOperations() public pure returns (uint) { return 10; } // 10 actions
+
+    // @notice The delay before voting on a proposal may take place, once proposed
+    function votingDelay() public pure returns (uint) { return 1; } // 1 block
+
+    // @notice The duration of voting on a proposal, in blocks
+    function votingPeriod() public pure returns (uint) { return 17280; } // ~3 days in blocks (assuming 15s blocks)
```

Comment:
create ability for timelock to change the voting period. Store votingPeriod as a variable

Reply:
It seems like the execution will revert when the target is the governorAlpha contract itself. I think we can use proxy to update to a new governor contract as an alternative